install/overlays/README.md

```diff
@@ -0,0 +1,98 @@
+
+# Kustomize Camel K
+
+Kustomize provides a declarative approach to the configuration customization of a Camel-K installation. Kustomize works either with a standalone executable or as a built-in to kubectl.
+
+Basic overlays are provided for easy usage.
+
+## HOW-TO
+
+### Initialize
+
+First create a new kustomization from the wanted version (kubernetes or openshift) in the repository:
+```sh
+kustomize create --resources https://github.com/apache/camel-k.git/install/overlays/kubernetes\?ref\=exp/kustomize_structure
+```
+
+You can also clone the camel-k repository and reference the local folder :
+```sh
+kubectl kustomize <path/to/localrepo/install/overlays/openshift | kubectl create -f -
+```
+
+To ensure the `IntegrationPlatform` custom resource is created add in the `kustomization.yaml`:
```

Comment:
Is it the only option we need to declare? in such case, I think it would be best to make a default value ootb.